ARIA heading role: aria-level is required (#39482)
mdn/content: files/en-us/web/accessibility/aria/reference/roles/heading_role/index.md

### Associated WAI-ARIA roles, states, and properties

- [`aria-level`](/en-US/docs/Web/Accessibility/ARIA/Reference/Attributes/aria-level)

>   - : The `aria-level` attribute specifies the heading level in the document structure. This attribute is required: authors must indicate the proper nesting level, to ensure elements with a role of heading are organized into a logical outline.

Reviewer: ```suggestion
  - : The `aria-level` attribute specifies the heading level in the document structure. This attribute is required: authors must indicate the proper nesting level to ensure elements with a `role` of `heading` are organized into a logical outline.
```

### Associated WAI-ARIA roles, states, and properties

- [`aria-level`](/en-US/docs/Web/Accessibility/ARIA/Reference/Attributes/aria-level)

>   - : The `aria-level` attribute specifies the heading level in the document structure. This attribute is required: authors must indicate the proper nesting level to ensure elements with a `role` of `heading` are organized into a logical outline. If the attribute is erroneously unspecified, then [browsers will use a fallback value of 2](https://w3c.github.io/aria/#document-handling_author-errors_states-properties).

Reviewer: ```suggestion
  - : The `aria-level` attribute specifies the heading level in the document structure. This attribute is required: authors must indicate the proper nesting level to ensure elements with a `role` of `heading` are organized into a logical outline. If the attribute is erroneously unspecified, then browsers will use a [fallback value of 2](https://w3c.github.io/aria/#authorErrorDefaultValuesTable).
```

>   - : The `aria-level` attribute specifies the heading level in the document structure. This attribute is required: authors must indicate the proper nesting level to ensure elements with a `role` of `heading` are organized into a logical outline. If the attribute is erroneously unspecified, then <a href="https://w3c.github.io/aria/#document-handling_author-errors_states-properties">browsers will use a fallback value of 2</a>.

Reviewer: ```suggestion
  - : The `aria-level` attribute specifies the heading level in the document structure. This attribute is required: authors must indicate the proper nesting level to ensure elements with a `role` of `heading` are organized into a logical outline. If the attribute is erroneously unspecified, then [browsers will use a fallback value of 2](https://w3c.github.io/aria/#document-handling_author-errors_states-properties).
```
